contrib/devtools/copyright_header.py

```diff
@@ -491,17 +491,28 @@ def file_has_hashbang(file_lines):
         return False
     return file_lines[0][:2] == '#!'
 
+def insert_shell_header(filename, file_lines, start_year, end_year):
```

Comment:
Why is this needed? Couldn't the function where you copied this from be used?

Reply:
Fixed.